sound/soc/bcm/audioinjector-pi-soundcard.c

```diff
@@ -158,9 +158,11 @@ static int audioinjector_pi_soundcard_probe(struct platform_device *pdev)
 			}
 	}
 
-	if ((ret = devm_snd_soc_register_card(&pdev->dev, card))) {
-		dev_err(&pdev->dev, "snd_soc_register_card failed (%d)\n", ret);
-	}
+	if ((ret = devm_snd_soc_register_card(&pdev->dev, card)))
+		return dev_err_probe(&pdev->dev, ret, "audioinjector_pi_soundcard_probe \n");
+
+	dev_err(&pdev->dev, "audioinjector_pi_soundcard_probe : successfully loaded\n");
```

Comment:
`dev_info` would be more appropriate. If you feel you just include the name of the function, use `%s` and `__func__`, but the name of the device is more user-friendly.

Reply:
Yes, I will change that error.